src/main/java/uk/gov/hmcts/opal/controllers/DraftAccountController.java

```diff
@@ -0,0 +1,73 @@
+package uk.gov.hmcts.opal.controllers;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.opal.dto.search.DraftAccountSearchDto;
+import uk.gov.hmcts.opal.entity.DraftAccountEntity;
+import uk.gov.hmcts.opal.service.DraftAccountServiceInterface;
+import uk.gov.hmcts.opal.service.opal.DraftAccountService;
+import uk.gov.hmcts.opal.service.opal.UserStateService;
+
+import java.util.List;
+
+import static uk.gov.hmcts.opal.util.HttpUtil.buildResponse;
+
+
+@RestController
+@RequestMapping("/api/draft-account")
+@Slf4j(topic = "DraftAccountController")
+@Tag(name = "DraftAccount Controller")
+public class DraftAccountController {
+
+    private final DraftAccountServiceInterface draftAccountService;
+
+    private final DraftAccountService opalDraftAccountService; // Might not be needed
+
+    private final UserStateService userStateService;
+
+    public DraftAccountController(@Qualifier("draftAccountService") DraftAccountServiceInterface draftAccountService,
```

Comment:
Should this @Qualifier reference the proxy ?

Reply:
No - we don't need either Legacy or Proxy services for draft accounts - they are purely an Opal-side artefact.